lib/assets/javascripts/views/plans/new.js

```diff
@@ -38,7 +39,7 @@ $(() => {
   const getMethod = jQueryForm => jQueryForm.attr('method');
 
   // When one of the autocomplete fields changes, fetch the available templates
-  const handleComboboxChange = () => {
+  const handleComboboxChange = debounce(() => {
     if ($('#plan_org_id').length && $('#plan_funder_id').length) {
       const validOrg = ($('#plan_org_id').val().trim().length > 0 || $('#plan_no_org').prop('checked'));
```

Comment:
please re-use functions, e.g isValidNumber from utils/isValidInputType for both const

Reply:
nice usage of debounce for improving communication with server.